integration-tests/universal/log_poller/helpers.go

```diff
@@ -18,35 +17,35 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	geth_types "github.com/ethereum/go-ethereum/core/types"
 	"github.com/rs/zerolog"
+	"github.com/smartcontractkit/wasp"
+
```

Comment:
btw. maybe that whole folder's content should be located in `integration-tests/actions` to make it more consistent with existing structure?

Reply:
Out of the scope of this PR but something we can definitely look into doing.